src/rebar3_eetcd_plugin.app.src

```diff
@@ -1,6 +1,6 @@
 {application,rebar3_eetcd_plugin,
              [{description,"Rebar3 plugin to generate client for eetcd"},
-              {vsn,"0.2.0"},
+              {vsn,"0.3.0"},
```

Comment:
I apologize for messing with the version but since the last release it has not been bumped so I needed to change it one way or another.